app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: 'author_id'
-  belongs_to :post, class_name: 'Post', foreign_key: 'post_id', counter_cache: :comments_counter
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id', counter_cache: :comments_count
```

Comment:
- [ ] Please remove the class_name attr from this association because the comments belongs to a post and this is the default name (unchanged as the user to author) of the `Post` model